source/brailleDisplayDrivers/handyTech.py

```diff
@@ -24,7 +25,36 @@
 import bdDetect
 import time
 import datetime
+from ctypes import windll

```

Comment:
I don't think you're using this here, right?

Reply:
ui was an unnecessary leftover from debugging. windll however is in use for registering the window message in line 42.

Reply:
Ah, thanks for clarifying. You can also access user32 from winUser, but I think this is ok as is.